controllers/comments.controllers.js

```diff
@@ -0,0 +1,17 @@
+const { fetchArticleById } = require('../models/articles.models');
+const { fetchCommentsByArticleId } = require('../models/comments.models')
+
+
+exports.getCommentsByArticleId = (req, res, next) => {
+    const { article_id } = req.params;
+
+    Promise.all([
+        fetchArticleById(article_id),
+        fetchCommentsByArticleId(article_id)
+    ])
+    .then((comments) => {
+        newComments = comments[1]
```

Comment:
you will have an array of comments

Reply:
Are we sending back all of the comments here?